src/messages.js

```diff
@@ -168,7 +169,6 @@ var warnings = {
   W089: "The body of a for in should be wrapped in an if statement to filter " +
     "unwanted properties from the prototype.",
   W090: "'{a}' is not a statement label.",
-  W091: "'{a}' is out of scope.",
```

Comment:
It looks like the convention here is to maintain properties for obsolete messages, but set them to `null`. I can only guess why (maybe to make it more obvious that the "skipped" message ID is intentional, maybe to make it easier to `git blame` the removal), but it's probably a good idea to stick with the convention for now.